packages/website/src/widgets/header/header.tsx

```diff
@@ -1,5 +1,5 @@
 import { Button, ButtonVariant } from "@lucky-parking/ui";
-import Logo from "src/shared/ui/logo";
+import Logo from "../../shared/ui/logo";
```

Comment:
It seems like when I run the test. Jest can read relative paths, but when we try the

```src/shared/ui/logo``` it cannot find module (absolute path)

I have tried configuring the jest.config.ts with

```
roots: ['<rootDir>/packages/website/src'],
moduleDirectories: ['<rootDir>/packages/website/src', 'node_modules']
 ```
 
 but could not get the absolute path working

Reply:
You can use the `pathsToModuleNameMapper` function from `ts-jest`.

https://kulshekhar.github.io/ts-jest/docs/getting-started/paths-mapping